docs/UserGuide.md

```diff
@@ -29,11 +29,50 @@ Example of usage:
 
 `todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
 
+### Display the list of all foods: `list`
+Displays the full list of food items stored in the fridge.
+
+Format: `list`
+
+* There should not be any descriptions after keyword `list`.  
+
+Example of usage: 
+
+`list`
+
+### Display the list of foods by category: `list <CATEGORY>`
+Displays the list of food items under specified category stored in the fridge.
+
+Format: `todo CATEGORY_NAME`
```

Comment:
I think you have missed out on replacing the word todo with list

Reply:
Yes thank you.